### ISSUE TITLE
"bad interpreter: Permission denied"

### ISSUE THREAD

**Issue by xuanruiqi** [ValveSoftware/steam-for-linux#7201]
#### Your system information

* Steam client version (build number or date): June 4, 2020
* Distribution (e.g. Ubuntu): Arch Linux
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:

I get this error when I try to run `/usr/bin/steam`:

    /usr/bin/steam: /usr/lib/steam/steam: /usr/bin/env: bad interpreter: Permission denied
    /usr/bin/steam: line 2: /usr/lib/steam/steam: Success

Running `sh /usr/bin/steam` is fine.
I have checked that `sh` and `bash` are symlinked and discovered by `env` correctly:

    $ type -a sh
    sh is /usr/bin/sh
	$ type -a bash
	bash is /usr/bin/bash
	$ getfacl /usr/bin/env
	getfacl: Removing leading '/' from absolute path names
	# file: usr/bin/env
	# owner: root
	# group: root
	user::rwx
	group::r-x
	other::r-x

Also, I am sure that noexec is not set on my file system. Other shebang scripts run without problem. In addition, other of my machines run the `/usr/bin/steam` script without issue. I'm out of ideas now, so I wonder what could be wrong here.

I use ZFS for my device; `/` and `/home` are datasets/subvolumes in my ZFS pool. I have no idea if this could be the culprit and it seems unlikely, but this might be it.

#### Steps for reproducing this issue:

1. Run `/usr/bin/steam`

**Comment by TTimo** [comment]
Arch maintains the steam installer for their distribution, this should probably be reported through their channels.

**Comment by TTimo** [comment]
CNR in any case (on Manjaro). The symlinks and trampoline scripts end up at `/usr/lib/steam/bin_steam.sh` - that's the script with the `/usr/bin/env` failing for you.

**Comment by xuanruiqi** [comment]
@TTimo : Hello, I did file a bug report on [Arch Linux bug tracker](https://bugs.archlinux.org/task/66953), and the Arch maintainers were unable to replicate or track down the bug. As a recourse they recommended that I open a bug here.

**Comment by muellphi** [comment]
I think I found it. I had exactly the same error with following conditions:
- Arch Linux
- FireJail 0.9.68
- Apparmor installed, loaded and firejail-default enforced
- I had 'apparmor' enabled in `~/.config/firejail/globals.local` (or `/etc/config/firejail/globals.local`)

This resulted in exactly the same error. I fixed it by creating following file:

`~/.config/firejail/steam.local`:

```
ignore apparmor
```

Now steam-runtime as well as steam-native works.
